### PR TITLE
 Add CLI install script 

### PR DESCRIPTION
**Pull request by Piccirello**
This script automatically fetches and installs the latest Doppler CLI. It can be downloaded from https://cli.doppler.com/install.sh

Documentation:
- [x] docs.doppler.com
- [x] readme.md
- [x] install.md
- [ ] notify customers (incl. changelog post)

After merge:
- [x] Update Cloudflare worker install.sh url to point to master

### CODE REVIEW

**Comment by Piccirello** [comment]
This install script needs to be documented:

- [ ] docs.doppler.com
- [ ] readme.md
- [ ] install.md
- [ ] notify customers (incl. changelog post)

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env sh
+
+set -e
```
Comment (bvallelunga) [inline scripts/install.sh:3]: Why is `set -e` needed?

**Review by bvallelunga — APPROVED** [review]

File: scripts/install.sh (hunk 1 above)
Comment (Piccirello) [reply scripts/install.sh:3]: `set -e` tells the shell to exit if any command returns non-zero. Without it, this script would continue executing even if the download failed:

```sh
curl -O https://cli.download.com/install.sh
ls install.sh
```

Comment (bvallelunga) [reply scripts/install.sh:3]: Makes sense, that is pretty cool!

**Pull request merged**